src/api/__tests__/apiHelpers.test.ts

```diff
@@ -241,4 +249,56 @@ describe('[api] helpers', () => {
       expect(result).toEqual({ error: 'Le refresh token est expiré' })
     })
   })
+
+  describe('handleGeneratedApiResponse', () => {
+    it('should return body without throwing error if status is 200', async () => {
```

Comment:
il n'y a pas que le 200 qui fonctionne de cette manière

je pense qu'on peut éviter de préciser "sans lancer une erreur" pour un cas nominal, sinon on devrait le dire partout

que penserais-tu de ?

```suggestion
    it('should return body when status is ok' async () => {
```